src/include/liburing.h

```diff
@@ -375,23 +375,27 @@ IOURINGINLINE void __io_uring_set_target_fixed_file(struct io_uring_sqe *sqe,
 	sqe->file_index = file_index + 1;
 }
 
+IOURINGINLINE void io_uring_initialize_sqe(struct io_uring_sqe *sqe) {
```

Comment:
Opening brace goes on the following line, as per the other functions in liburing.